sdk/openai/azure-openai/samples/images_aoai_quickstart.py

```diff
@@ -0,0 +1,79 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: images_aoai_quickstart.py
+
+DESCRIPTION:
+    This sample demonstrates how to get started generating images with the Azure OpenAI SDK for Python.
+
+USAGE:
+    python images_aoai_quickstart.py
+
+    Before running the sample:
+
+    pip install openai
+    pip install requests
+    pip install pillow
+
+    Set the environment variables with your own values:
+    1) AZURE_OPENAI_KEY - your Azure OpenAI API key.
+    2) AZURE_OPENAI_ENDPOINT - the endpoint to your Azure OpenAI resource.
+    3) AZURE_OPENAI_IMAGE_DEPLOYMENT - the deployment name you chose when deploying your model.
+"""
+
+import os
+os.environ["AZURE_OPENAI_ENDPOINT"] = os.getenv("AZURE_OPENAI_SWEDENCENTRAL_ENDPOINT")
+os.environ["AZURE_OPENAI_IMAGE_DEPLOYMENT"] = "dall-e-3"
+
+def images_aoai_quickstart() -> None:
+    import os
+    from openai import AzureOpenAI
+    import requests
+    from PIL import Image
+    import json
+    from azure.identity import DefaultAzureCredential, get_bearer_token_provider
+
+    token_provider = get_bearer_token_provider(
+        DefaultAzureCredential(), "https://cognitiveservices.azure.com/.default"
+    )
+
+    client = AzureOpenAI(
+        azure_ad_token_provider=token_provider,
+        azure_endpoint=os.environ["AZURE_OPENAI_ENDPOINT"],
+        api_version=os.environ["API_VERSION_GA"]
+    )
+
+    result = client.images.generate(
+        model=os.environ["AZURE_OPENAI_IMAGE_DEPLOYMENT"],
+        prompt="a close-up of a bear walking throughthe forest",
+        n=1
+    )
+
+    json_response = json.loads(result.model_dump_json())
```

Comment:
I don't think we need to convert this to JSON, we can work with the type itself.